api/taskcluster/webhook.go

```diff
@@ -223,15 +223,24 @@ func shouldProcessStatus(log shared.Logger, processAllBranches bool, status *sta
 
 func parseTaskclusterURL(targetURL string) (rootURL, taskGroupID, taskID string) {
 	if matches := inspectorURLRegex.FindStringSubmatch(targetURL); len(matches) > 2 {
-		return matches[1], matches[2], ""
-	}
-	if matches := taskURLRegex.FindStringSubmatch(targetURL); len(matches) > 2 {
+		rootURL = matches[1]
+		taskGroupID = matches[2]
+	} else if matches := taskURLRegex.FindStringSubmatch(targetURL); len(matches) > 2 {
 		if len(matches) > 2 {
```

Comment:
Maybe a Go-newb question, but don't we already check `len(matches) > 2)` on the prev line? If so, seems like the `else` below won't ever execute?

Reply:
Fixed.